pkg/cmd/cli/backup/create.go

```diff
@@ -33,11 +33,11 @@ import (
 	"github.com/heptio/ark/pkg/cmd/util/output"
 )
 
-func NewCreateCommand(f client.Factory) *cobra.Command {
+func NewCreateCommand(f client.Factory, use string) *cobra.Command {
 	o := NewCreateOptions()
 
 	c := &cobra.Command{
```

Comment:
It would be nice to document that `ark create backup` and `ark backup create` are synonymous, so it's not confusing to a user why both exist. (for all the commands)

Reply:
Do you have a suggestion for how to best do this? Maybe just in the help for `ark create` and `ark get` to say e.g. `Create ark resources. These commands are synonymous with the individual resource-specific create commands, such as 'ark backup create'.`?

Reply:
I'm thinking maybe just in the `Long` help text for the `ark` command, we add an explanation that we support both forms? Since that's where if a user asks for help, they'll see both the `backup`, etc. commands as well as `get`, etc.

Reply:
I can do that